test/test_images/latencymako_sender_receiver/main.go

```diff
@@ -404,12 +401,7 @@ func processVegetaResult(vegetaResults <-chan *vegeta.Result, doneCh chan<- stru
 }
 
 func processReceiveEvent(event cloudevents.Event) {
-	et, err := ptypes.TimestampProto(event.Time())
```

Comment:
Good catch! `event.Time()` should always be "timestamp of the occurrence" according to the CloudEvents spec.